src/tutorials/using-a-pathogen-repo/running-an-ingest-workflow.rst

```diff
@@ -0,0 +1,249 @@
+
+In this tutorial, you will run the ingest workflow of our `Zika repository <https://github.com/nextstrain/zika>`_ and view outputs on your computer.
+You will have a basic understanding of how to run ingest workflows for other pathogens and a foundation for understanding how to customize ingest workflows.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+=============
+
+1. :doc:`Install Nextstrain </install>`. These instructions will install all of the software you need to complete this tutorial.
+
+Download the Zika repository
+============================
+
+All pathogen ingest workflows are stored in :term:`pathogen repositories<pathogen repository>` (version-controlled folders) to track changes over time.
+Download the `Zika repository <https://github.com/nextstrain/zika>`_.
+
+.. code-block::
+
+    $ git clone https://github.com/nextstrain/zika
+    Cloning into 'zika'...
+    [...more output...]
+
+When it's done, you'll have a new directory called ``zika/``.
+
+Run the default workflow
+========================
+
+The zika :term:`ingest workflow` uses the `NCBI Datasets CLI tools <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/>`_
+to download public data and uses a combination of :doc:`augur curate <augur:usage/cli/curate/index>` and other data manipulation tools to curate
+the downloaded data into a format suitable for :term:`phylogenetic workflows <phylogenetic workflow>`.
+
+1. Change directory to the Zika pathogen repository downloaded in the previous step
+
+.. code-block::
+
+    $ cd zika
+
+2. Run the default ingest workflow with the :term:`Nextstrain CLI`.
+
+.. code-block::
+
+    $ nextstrain build ingest
+    Using profile profiles/default and workflow specific profile profiles/default for setting default command line arguments.
+    Building DAG of jobs…
+    [...a lot of output...]
+
+This should take just a few minutes to complete.
+There should be two final output files:
+
+* ``ingest/results/metadata.tsv``
+* ``ingest/results/sequences.fasta``
+
+The output files should have the same data formats as the public data files hosted by Nextstrain, available at:
+
+* https://data.nextstrain.org/files/workflows/zika/metadata.tsv.zst
+* https://data.nextstrain.org/files/workflows/zika/sequences.fasta.zst
+
+Your results may have additional records depending on whether new data has been released since the public data files were last uploaded.
+
+
+Configuring the ingest workflow
+===============================
+
+Now that you've seen the default outputs of the ingest workflow, you can try configuring the ingest workflow to change the outputs.
+If you want to see the uncurated NCBI Datasets data to decide what changes you would like to make to the workflow, you can run the following:
+
+.. include:: ../../snippets/uncurated-ncbi-dataset.rst
+
+We'll walk through an example custom config to include an additional column in the curated output.
```

Comment:
[minor] - can you add a small subheading here to demarcate this from the snippet contents?

Reply:
Added subheading in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/6019d504aaa114e8c928c27faa3c676fd6a71d18